UIKITDZ/UIKITDZ/ViewController.swift

```diff
@@ -7,11 +7,45 @@
 
 import UIKit
 
-/// ViewController
-class ViewController: UIViewController {
-
+/// Контроллер входа , проверяет имя и логин, если корректны - открывает экран кафемарио
+final class ViewController: UIViewController {
+    @IBOutlet private weak var enterEmailTextField: UITextField!
+    @IBOutlet private weak var enretPasswordTextField: UITextField!
+    
+    private var visitors = Payment()
```

Comment:
```suggestion
    private var  payment  = Payment()
```
переменная так же как и модель